### PR TITLE
Improve admin area on mobile

### PR DESCRIPTION
**Pull request by jezemery**
Make the checking in process a little more mobile friendly with anchors, and hiding irrelevant content

### CODE REVIEW

**Comment by asgrim** [comment]
@jezemery tests failed though as expectations were not updated for generated URLs

File: public-src/phphants/scss/admin.scss [hunk 1]
```diff
@@ -47,6 +47,17 @@ a.link-button {
...
+    @media (max-width: 320px) {
+        display: inline-block;
+        margin: 5px auto 0 auto;
+        width: 100%;
+    }
+}
+.hide-mobile {
+    @media (max-width: 320px) {
+        display: none;
+        visibility: hidden;
```
Comment (lboynton) [inline public-src/phphants/scss/admin.scss]: Just wondering, why use both `display` and `visibility` properties?

File: src/App/Action/Account/Meetup/CancelCheckInAction.php [hunk 2]
```diff
@@ -56,6 +56,6 @@ public function __invoke(Request $request, Response $response, callable $next =
             $meetup->cancelCheckIn($user);
         });
 
-        return new RedirectResponse($this->urlHelper->generate('account-meetup-view', ['uuid' => $meetup->getId()]));
+        return new RedirectResponse($this->urlHelper->generate('account-meetup-view', ['uuid' => $meetup->getId()]).'#attendees');
```
Comment (lboynton) [inline src/App/Action/Account/Meetup/CancelCheckInAction.php]: Fourth param of the URL helper is the URI fragment: https://zend-expressive.readthedocs.io/en/latest/features/helpers/url-helper/

:slightly_smiling_face:

File: public-src/phphants/scss/admin.scss (hunk 1 above)
Comment (jezemery) [reply public-src/phphants/scss/admin.scss]: Not sure, I'll take this out ;)

File: src/App/Action/Account/Meetup/CancelCheckInAction.php (hunk 2 above)
Comment (jezemery) [reply src/App/Action/Account/Meetup/CancelCheckInAction.php]: I'll take a look and fix it. Thanks :)

Comment (lboynton) [reply src/App/Action/Account/Meetup/CancelCheckInAction.php]: @jezemery I think changing this will fix the tests too :slightly_smiling_face:

Comment (jezemery) [reply src/App/Action/Account/Meetup/CancelCheckInAction.php]: @lboynton the version of expressive we're using doesn't support this - https://github.com/zendframework/zend-expressive/blob/1.0.0/doc/book/features/helpers/url-helper.md 

Ideas?

Comment (lboynton) [reply src/App/Action/Account/Meetup/CancelCheckInAction.php]: @jezemery Despair?

I guess in that case do what you've done and append '#attendees' to the line here: 

- https://github.com/phphants/phph-site/blob/256c31eac92919c95065367b9044a23e3f2e5126/test/AppTest/Action/Account/Meetup/CancelCheckInActionTest.php#L54
- https://github.com/phphants/phph-site/blob/256c31eac92919c95065367b9044a23e3f2e5126/test/AppTest/Action/Account/Meetup/CheckInUserActionTest.php#L53

Comment (lboynton) [reply src/App/Action/Account/Meetup/CancelCheckInAction.php]: Aren't we using version 2 though? https://github.com/phphants/phph-site/blob/master/composer.json#L12

Comment (jezemery) [reply src/App/Action/Account/Meetup/CancelCheckInAction.php]: You're not wrong, but when I go and find that function, there's no mention of features available in v2.0 😕

Comment (lboynton) [reply src/App/Action/Account/Meetup/CancelCheckInAction.php]: Ah, it was added in version 3 of zend-expressive-helpers: https://github.com/zendframework/zend-expressive-helpers/blob/develop/CHANGELOG.md#300---2016-01-11

Looking at the changelog, we should be good to upgrade I think.

Comment (jezemery) [reply src/App/Action/Account/Meetup/CancelCheckInAction.php]: That's weird... it's in the documentation for v2.0...

File: src/App/Action/Account/Meetup/CancelCheckInAction.php [hunk 3]
```diff
@@ -56,6 +56,6 @@ public function __invoke(Request $request, Response $response, callable $next =
             $meetup->cancelCheckIn($user);
         });
 
-        return new RedirectResponse($this->urlHelper->generate('account-meetup-view', ['uuid' => $meetup->getId()]));
+        return new RedirectResponse($this->urlHelper->generate('account-meetup-view', ['uuid' => $meetup->getId()], [], 'attendees'));
```
Comment (lboynton) [inline src/App/Action/Account/Meetup/CancelCheckInAction.php]: Just the line length is too long now :smile:

Comment (jezemery) [reply src/App/Action/Account/Meetup/CancelCheckInAction.php]: 😱

**Comment by jezemery** [comment]
@lboynton happy to merge this now? :)

**Comment by lboynton** [comment]
@jezemery Yeah if everything is still working :smile:

**Comment by jezemery** [comment]
I mean, it passes tests, and @asgrim wrote the tests... So if it doesn't, it's his fault right? 😆

**Pull request merged**